crates/viewer/re_selection_panel/src/selection_panel.rs

```diff
@@ -211,8 +211,8 @@ impl SelectionPanel {
                 }
 
                 ui.list_item_flat_noninteractive(
-                    PropertyContent::new("Archetype field")
-                        .value_text(component_descriptor.archetype_field_name()),
+                    PropertyContent::new("Component")
+                        .value_text(component_descriptor.component.as_str()),
```

Comment:
We already does a desctruction of `ComponentDescriptor` and intentionally ignore `component` in it, and then extract it explicitly here anyways… do one or the other